snapcraft/main.py

```diff
@@ -116,6 +123,27 @@ def _get_version():
         return 'devel'
 
 
+def _scaffold_examples(directory):
+    logger.debug("Copying examples to {}".format(directory))
+    dest_dir = os.path.abspath(directory)
+
+    # If dest_dir doesn't exist, we dump all examples in it.
+    # If it does exist, we dump them into a subdirectory
+    try:
+        shutil.copytree(get_examplesdir(), dest_dir)
+    except FileExistsError:
+        # don't event try to copy if the dest exists already
+        if not os.path.isdir(dest_dir):
+            raise NotADirectoryError("{} is a file, can't be used as a "
+                                     "destination".format(dest_dir))
+        dest_dir = os.path.join(dest_dir, _SNAPCRAFT_TOUR_DIR)
+        shutil.copytree(get_examplesdir(), dest_dir)
+
+    print("Snapcraft tour initialized in {}.\n"
```

Comment:
I think this should be logger.info, but we have some inconsistencies here in how to handle errors and messages in main. That's a bug for the next milestone.